config.go

```diff
@@ -48,16 +43,10 @@ func loadConfig(file []byte, cfg *config) error {
 			return fmt.Errorf("parse issuer url %d: %w", i, err)
 		}
 		cfg.Issuer[i].URL = parsed
-		for ii, c := range cfg.Issuer[i].Client {
-			if c.ClientID == "" {
+		for ii, c := range cfg.Issuer[i].Clients {
+			if c.ID == "" {
 				return fmt.Errorf("issuer %s client %d must set clientID", parsed, ii)
 			}
-			if len(c.ClientSecret) == 0 {
-				return fmt.Errorf("issuer %s client %d must set at least one clientSecrets", parsed, ii)
-			}
-			if len(c.RedirectURL) == 0 {
-				return fmt.Errorf("issuer %s client %d must set at least one redirectURLs", parsed, ii)
-			}
```

Comment:
Would be nice to validate early still; could this call IsValidClientID etc at the end?

Reply:
redirect URL isn't always needed - I'll add some better validation to the upstream library.